exercises/isogram/isogram_test.rb

```diff
@@ -77,6 +77,6 @@ def test_duplicated_character_in_the_middle
 
   def test_bookkeeping
     skip
-    assert_equal 3, BookKeeping::VERSION
+    assert_equal 4, BookKeeping::VERSION
```

Comment:
Ah yes, removed the query part that is common in languages where the question mark cannot be used.  So it is a good version bump.